backend/models/questionSchema.js

```diff
@@ -0,0 +1,19 @@
+var mongoose = require("mongoose");
+
+var questionSchema =  mongoose.Schema({
+    id : { type: Number, required: true, unique: true },
+    name: { type: String, required: true },
+    question: { type: String, required: true },
+    answer: { type: String, required: true },
+    falseAnswers : [String]
+  });
+
+var packSchema = mongoose.Schema({
+  id : { type: Number, required: true, unique: true },
```

Comment:
if you guys plan on using this ID (i don't think its necessary) make sure you give it a default value function. Search it up on mongoose Schema docs.